noxfile.py

```diff
@@ -331,8 +336,8 @@ def _get_standalone_apps_dirs():
 @nox.session(python=PYTHON_VERSIONS)
 @nox.parametrize(
     "install_cmd",
-    PLUGINS_INSTALL_COMMANDS,
-    ids=[" ".join(x) for x in PLUGINS_INSTALL_COMMANDS],
+    (INSTALL_COMMAND,),
```

Comment:
you can clean this up. this is no longer a list. (you are making it a list lf 1.
you can just take it out of the parameterization.

Reply:
sure, will do!

Reply:
cleaned up